source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Publisher: Unprocessed fact-check emails'
+section: Icinga alerts
+subsection: Email alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-07-02
+review_in: 6 months
+---
+
+### Unprocessed fact-check emails
+
+This means there are unprocessed emails in the fact-check inbox
+(govuk-fact-check@digital.cabinet-office.gov.uk).
+Publisher has a [script (`mail_fetcher`)](script) that checks the inbox
+[every 5 minutes][schedule] and [processes all emails present][process].
+
+If emails are not processed or deleted, this alert is triggered. The level of
+the [alert][] depends on the number of emails present - which was chosen
+arbitrarily and is currently:
+- warning with 1 email
+- critical with 2 or more emails
+
+### Log into the fact-check inbox
+
+If this alert is triggered, we want to log into the fact-check inbox and
+investigate why the emails were not processed and deleted.
+
+The password to log into the govuk-fact-check@digital.cabinet-office.gov.uk
+inbox can be found by running:
+
+```bash
+PASSWORD_STORE_DIR=~/govuk/govuk-secrets/pass/2ndline pass google-accounts/govuk-fact-check@digital.cabinet-office.gov.uk
```

Comment:
This account isn't currently the _production_ account until we've done https://trello.com/c/2ZckRO3G/1733-switch-fact-check-address-away-from-alphagovcouk

Instead it's an `@alphagov.co.uk` account (I'm not actually sure if the credentials are in the password store, I think it's in Hieradata secrets).